static/js/src/entry/account/reset-password/components/LinkListItem.vue

```diff
@@ -0,0 +1,42 @@
+<template>
+  <reset-password-provider v-slot="{ pwResetSuccess, pwResetFailure, pwReset }">
+    <link-list-item>
+      <template v-slot:main>
+        <button class="c-link-button" @click="pwReset(pwResetGaLabel)">
+          Reset your password
+        </button>
+      </template>
+      <template v-slot:extra>
+        <p v-if="pwResetSuccess" class="t-size-xs has-text-gray">
```

Comment:
We changed all `has-text-gray` to `has-text-gray-dark` after our contrast bug overhaul. I guess eventually we could deprecate `has-text-gray` altogether